notebooks/data analysis/Pathfinder_system_abilities_attributes.ipynb

```diff
@@ -0,0 +1,1915 @@
+     },
+     "execution_count": 113,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ac.head()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 119,
+   "outputs": [],
+   "source": [
+    "ac = ac.replace(\"\", np.NAN)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 126,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Null values:\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": "1147"
+     },
+     "execution_count": 126,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "print(\"Null values:\")\n",
+    "ac[\"details\"].isnull().sum()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 123,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "all-around vision                       25\nfed by metal                             6\n(21 with shield raised)                  2\n(25 with shield raised)                  2\n+1 status vs. evil creatures             2\n(22 when broken); construct armor        2\n(20 with shield raised)                  2\n22 with shield raised                    2\n(18 with shield raised)                  1\n(32 when broken); construct armor        1\n(26 when broken); construct armor        1\n(31 with guard raised)                   1\n(26 with shield raised)                  1\n(27 without alien vestment)              1\n(29 with shields raised)                 1\n(35 when broken); construct armor        1\n(22 against prey; see Hunt Prey)         1\n(10 from inside the Dimensional Pit)     1\nor 36 vs. non-magical                    1\n(14 when broken); construct armor        1\n(13 when broken); construct armor        1\n(22 with shield raised)                  1\n(15 when broken); construct armor        1\n(12 when broken); construct armor        1\nName: details, dtype: int64"
+     },
+     "execution_count": 123,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ac[\"details\"].value_counts()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Można olać całość i uwzglednic jedynie **value**\n",
+    "\n",
+    "Przynajmniej tak mi się wydaje\n",
```

Comment:
Taaa, tylko 29 wartości i to trudnych w analizie. Można olać

Reply:
XD Jak ty to policzyłaś?

Reply:
XD
Jak ty to policzyłaś?